packages/atoms/booker/Booker.tsx

```diff
@@ -0,0 +1,12 @@
+import { Booker as BookerComponent, BookerProps } from "@calcom/features/bookings/Booker";
```

Comment:
We decided that this PR should first be about making a new booker component, later (in a new PR) we'll be turning this into a reuable package. So anything in `packages/atoms` can be ignored for now, it's a veeery rough and incomplete basis. Initial challenges I could already think about are summed up in todo's in this file.